docs/analytics/types.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Supported Types
+---
+
+`parquet` tables support most, but not all, built-in Postgres types. If a type is not listed below, it is not
```

Comment:
Can we add a sentence here to say "if you need a specific type, please contact us in Slack(link) or by email @ ...

Just want to make sure users feel the door is open to coming forward to us with their requirements

Reply:
I would prefer people just create a github issue, and our users should know to do that